README.md

```diff
@@ -1,2 +1,19 @@
 # code-review-bot
-code-review-bot
+
+Install Go Task
+
+```sh
+brew install go-task
+```
+
+Install dependencies
```

Comment:
Consider adding troubleshooting steps or common issues users may encounter during the installation or execution steps to enhance the guide.